src/components/Date/DateField.js

```diff
@@ -11,20 +11,28 @@ import { isToday, isYesterday } from '../../util/datetime';
 
 type Props = {
     date: string,
+    dateFormat?: Object,
+    includeCommas?: boolean,
     relative: boolean,
     intl: any
 };
 
-const DateField = ({ date, intl, relative = true }: Props) => {
+const DateField = ({ date, dateFormat, includeCommas, intl, relative = true }: Props) => {
     const d = new Date(date);
     if (relative && isToday(d)) {
         return <FormattedMessage {...messages.today} />;
     } else if (relative && isYesterday(d)) {
         return <FormattedMessage {...messages.yesterday} />;
     }
 
-    const formattedDate = intl.formatDate(d, { weekday: 'short', month: 'short', year: 'numeric', day: 'numeric' });
-    return <span>{formattedDate.replace(/,/g, '')}</span>;
+    let formattedDate = intl.formatDate(d, dateFormat);
+    formattedDate = includeCommas ? formattedDate : formattedDate.replace(/,/g, '');
+    return <span>{formattedDate}</span>;
+};
+
+DateField.defaultProps = {
```

Comment:
I could switch these to defaults in the object destructuring above if that is preferred for a stateless functional component.

Reply:
yes